packages/main/src/List.hbs

```diff
@@ -16,6 +16,13 @@
 	@ui5-_selection-requested={{onSelectionRequested}}
 	@ui5-_focus-requested={{onFocusRequested}}
 >
+	<ui5-busy-indicator
+		id="{{_id}}-busyIndicator"
+		delay="{{loadingDelay}}"
+		?active="{{loading}}"
+		?inert="{{loading}}"
```

Comment:
I think that you don't need `inert` in this case. Otherwise loading list cannot be focused neither with the keyboard, nor with the mouse.
If you remove it, it will be possible. Just check whether screen readers announce it properly.

Reply:
Thanks for the review,

The screen readers says "zero percent" as if it's loading and we expect it to get to 100%. I don't know if this is expected.

Reply:
@dobrinyonkov is the expert here. The rest looks good to me :)